### ISSUE TITLE
First Changes: Essay Hero Image Styling

### ISSUE THREAD

**Issue by vhellstein** [DenverArtMuseum/biophilia-pub#27]

**Comment by vhellstein** [comment]
- [ ] Title and author to the left, image to the right. Like this: see this essay from Emergence Magazine [](https://emergencemagazine.org/essay/when-you-could-hear-the-trees/)

- [ ] background of left side text: accent green with white title OR off white with black text

@mpopke will need to crop these--see Darrin's notes that he sent us for the cropping he prefers.
- [ ] Natural Analogs: PTL-24672 
- [ ] Natural Systems: PTL-24699~6
- [ ] Topophilia: 2017.89–91 [bottom left corner of far left object-- @mpopke there might be a better image to crop in Piction

**Comment by 1000camels** [comment]
Cropping the image yourself and adding it as another figure image is a good idea

**Comment by 1000camels** [comment]
Are the two colour variations rollover states?

**Comment by vhellstein** [comment]
No, it was just two options. We'll see what Darrin thinks, and we'll need to home in on the right green accent color throughout.
